<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListMylistForm</class>
 <widget class="QWidget" name="ListMylistForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="table_list_mylist">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>431</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
